### PR TITLE
Add gl_* cvar info

### PR DESCRIPTION
**Pull request by Leo40Git**
Some of this should probably be split into an FAQ section...

### CODE REVIEW

**Comment by mastercoms** [comment]
I think it should be mentioned that they're harmless and can be ignored.

**Comment by Leo40Git** [comment]
oops, added

File: docs/next_steps/troubleshoot.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ If TF2 is crashing to desktop after a custom map loading / after picking a class
 
 ## Item panels are taking too long to load
 
-Set `tf_time_loading_item_panels 0.0005`.
+Set `tf_time_loading_item_panels` to `0.0005`.
```
Comment (mastercoms) [inline docs/next_steps/troubleshoot.md]: This change makes it harder to copy paste.

Comment (Leo40Git) [reply docs/next_steps/troubleshoot.md]: fixed, https://github.com/mastercoms/mastercomfig/pull/126/commits/0d39d04e654d630913745fa39ddedaa81c3fe2fc

**Comment by mastercoms** [comment]
Thanks for your contribution to mastercomfig!

**Pull request merged**